app/serializers/event.js

```diff
@@ -9,5 +10,14 @@ export default ApplicationSerializer.extend(CustomPrimaryKeyMixin, {
     topic     : 'event-topic',
     subTopic  : 'event-sub-topic',
     copyright : 'event-copyright'
+  },
+  serialize(snapshot, options) {
```

Comment:
Why are we omitting these? 
And if you want to ignore some attributes, you use readonly option in the model. 

hello: attr('string', { readonly: true}) ;

I think I added it